platoon_strategic_IHP/config/parameters.yaml

```diff
@@ -0,0 +1,19 @@
+maxPlatoonSize: 10
+algorithmType: 0
+mvr_duration: 15
+timeHeadway: 2.5
+standStillHeadway: 12.0
+maxAllowedJoinTimeGap: 15.0
+maxAllowedJoinGap: 90.0
+desiredJoinTimeGap: 4.0
+desiredJoinGap: 30.0
+waitingStateTimeout: 25.0
+cmdSpeedMaxAdjustment: 10.0
+lowerBoundary: 1.6
+upperBoundary: 1.7
+maxSpacing: 4.0
+minSpacing: 3.9
+minGap: 22.0
+maxGap: 32.0
+maxCrosstrackError: 0.8
+vehicleLength: 5.0
```

Comment:
Vehicle length is already available from the VehicleConfigParams file in the carma-config repo.  Duplicating it here will cause configuration management problems.